crowbar_framework/app/controllers/machines_controller.rb

```diff
@@ -88,10 +88,19 @@ def role
 
   add_help(:rename, [:id])
   def rename
-    @machine.alias = params[:alias]
+    begin
+      @machine.alias = params[:alias]
+    rescue StandardError => e
+      alias_error = e.message
+    end
 
     respond_to do |format|
-      if @machine.save
+      if !alias_error.nil?
```

Comment:
@MaximilianMeister I added it there just in case, but as I was saying, in testing that did not show issues